packages/components/addon/components/hds/form/utils/getElementId.js

```diff
@@ -0,0 +1,14 @@
+import { guidFor } from '@ember/object/internals';
```

Comment:
[question] can you explain why this is in a utils folder inside of the form components folder, instead of a standard helper (`ember generate helper elementId` or something)?

Reply:
@MelSumner helpers are meant to be used from within `.hbs` templates, not in `.js` files, correct?

Reply:
yep! what @Dhaulagiri says (and doesn't make sense to move it up in the hierarchy, because is very specific to the form controls needs).